ui-tests/package.json

```diff
@@ -66,5 +66,8 @@
     "ignorePatterns": [
       "cypress/support/index.d.ts"
     ]
+  },
+  "devDependencies": {
+    "cy-verify-downloads": "0.0.7"
```

Comment:
Please, move this to the dependencies to make sure we are always able to execute the tests

Reply:
Done